public/app/core/services/echo/init.ts

```diff
@@ -168,8 +167,8 @@ async function initRudderstackBackend() {
       writeKey: config.rudderstackWriteKey,
       dataPlaneUrl: config.rudderstackDataPlaneUrl,
       user: contextSrv.user,
-      sdkUrl: config.rudderstackSdkUrl,
-      configUrl: configUrl,
+      sdkUrl,
+      configUrl: config.rudderstackConfigUrl,
```

Comment:
Before this PR the `configUrl` was the `config.rudderstackSdkUrl` but now it is the `config.rudderstackConfigUrl`. Is that correct?

Reply:
Yep, this is a corrected mistake from a previous PR! "configUrl" and "sdkUrl" were a little too similar for my brain last week 😓